build.gradle

```diff
@@ -95,13 +104,13 @@ subprojects {
     apply plugin: 'maven-publish'
     apply plugin: 'checkstyle'
 
-    if (it.name != 'crypto-android') {
+    if (it.name != 'crypto-android' && it.name != 'android-utils') {
```

Comment:
Can we add 'isAndroid' flag to module and use it in script? It would allow us to avoid hardcoded.

Reply:
good idea, will try to implement

Reply:
Unfortunately - we can not. Gradle builds projects from top, so first project evaluated is 'parent', which defines how to build child projects in `subprojects` closure. So we need a variable from non-evaluated project to evaluate it, while variables are accessible only after evaluation of project, and we're stuck.